package.json

```diff
@@ -3,13 +3,19 @@
   "version": "1.1.2",
   "scripts": {
     "test": "make test",
-    "release": "release"
+    "release": "release",
+    "prepublish": "make build"
   },
-  "main": "index",
+  "main": "dist/door-umd.js",
+  "jsnext:main": "dist/door-es6.js",
```

Comment:
It's coming from them, but it seems that they want this to be a standard in the future. There has been rumblings from the Node team about using an ES2015 module system alongside what they already have once the loader spec has solidified and Chrome has implemented it. Since rollup is early to set this, it'll probably be the one to use later: https://github.com/rollup/rollup/wiki/jsnext:main